addons/hr_recruitment/data/hr_recruitment_data.xml

```diff
@@ -16,12 +16,89 @@
         <field name="lang">${object.partner_id and object.partner_id.lang or ''}</field>
         <field name="model_id" ref="hr_recruitment.model_hr_applicant"/>
         <field name="user_signature" eval="False"/>
-        <field name="body_html"><![CDATA[
-<p>Dear ${object.partner_name or 'applicant'},</p>
-<p>Thank you for your enquiry.<br />
-Unfortunately, your profile does not match with our needs</p>
-<p>If you have any questions, please let us know.</p>
-<p>Best regards,</p>]]></field>
+        <field name="body_html" type="html">
+<table cellspacing="0" cellpadding="0" width="100%" style="font-family: 'Helvetica'; background-color: #EEE; color:#666666; border-collapse: collapse;">
+    <tr>
+        <td valign="top" align="center">
+            <table cellspacing="0" cellpadding="0" width="600" style="margin: 0 auto; width: 600px;">
+                <tr>
+                    <td>
+                        <table cellspacing="0" cellpadding="0" width="100%">
+                            <tr>
+                                <td valign="middle" align="left" style="padding:15px">
+                                    <img width="90" src="https://www.odoo.com/openerp_website/static/src/img/2016/logos/odoo_logo_small.png" style="width:80px;" />
+                                </td>
+                            </tr>
+                        </table>
+                        <table cellspacing="0" cellpadding="0" width="100%" bgcolor="#fff" style="background-color:#fff;">
+                            <tr>
+                                <td width="100%" style="padding: 15px;">
+                                    <table cellspacing="0" cellpadding="0" width="100%">
+                                        <tr>
+                                            <td style="margin-bottom: 30px;">
+                                                <p style="font-size:18px;">Dear ${object.partner_name or 'applicant'},</p>
```

Comment:
Maybe convert to qweb while at it, since the templates are being rewritten either way?